app/serializers/activitypub/vote_serializer.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+class ActivityPub::VoteSerializer < ActiveModel::Serializer
+  class NoteSerializer < ActiveModel::Serializer
+    attributes :id, :type, :content, :attributed_to,
+               :in_reply_to, :to
+
```

Comment:
The examples in the spec use `name` and not `content`. It also doesn't set a `type`: https://www.w3.org/TR/activitystreams-vocabulary/#dfn-question

I guess one upshot of doing that is to make sure you do not flood software that does not support this with vote `Note`s.

Reply:
I believe Misskey sends them as content? cc @syuilo

Also your upshot is also a downshot. Since the content is the poll option text, it works for the human observers even if it's not officially interpreted as a vote.

Reply:
Misskey is using `name` for now, but they don't have `type` property. Should we add `type` property or use `content` instead of `name`?

<details>
<summary>sample json</summary>
<pre><code>{
  "@context": [
    "https://www.w3.org/ns/activitystreams",
    "https://w3id.org/security/v1",
    { "Hashtag": "as:Hashtag" }
  ],
  "type": "Question",
  "id": "https://emoji.msky.work/questions/5c7bfbc0d4ab42731132b980",
  "actor": "https://emoji.msky.work/users/5be70e1033638b16ae9c6469",
  "content": "1 choice poll test.",
  "oneOf": [
    { "name": "test", "_misskey_votes": 6 }
  ]
}
</code></pre>
</details>

Reply:
@acid-chicken I mean poll votes, do the votes use content or name?

Reply:
@Gargron Voting activities are represented as Note, and they're using `content` in Misskey. (I'm sorry for miss reading!)

<details>
<summary>sample json</summary>
<pre><code>{
  "id": "https://misskey.m544.net/notes/5c7c152d34671559e850a1f8/activity",
  "actor": "https://misskey.m544.net/users/5bf0009a2d98942caa53e39e",
  "type": "Create",
  "published": "2019-03-03T17:55:57.525Z",
  "object": {
    "id": "https://misskey.m544.net/notes/5c7c152d34671559e850a1f8",
    "type": "Note",
    "attributedTo": "https://misskey.m544.net/users/5bf0009a2d98942caa53e39e",
    "summary": null,
    "content": "&lt;p&gt;&lt;span&gt;1&lt;/span&gt;&lt;/p&gt;",
    "_misskey_content": "1",
    "published": "2019-03-03T17:55:57.525Z",
    "to": [
      "https://emoji.msky.work/users/5be70e1033638b16ae9c6469"
    ],
    "cc": [],
    "inReplyTo": "https://emoji.msky.work/notes/5c7c1521ec5f6f5c1af0597e",
    "attachment": [],
    "sensitive": false,
    "tag": [
      {
        "type": "Mention",
        "href": "https://emoji.msky.work/users/5be70e1033638b16ae9c6469",
        "name": "@acid_chicken@emoji.msky.work"
      }
    ]
  },
  "to": [
    "https://emoji.msky.work/users/5be70e1033638b16ae9c6469"
  ],
  "cc": []
}
</code></pre>
</details>